Makefile

```diff
@@ -26,11 +26,13 @@ fixup_locale:
 snag_docs: Documentation/man1/git-filter-repo.1 Documentation/html/git-filter-repo.html
 
 Documentation/man1/git-filter-repo.1:
-	mkdir -p man1
+	mkdir -p Documentation/man1
+	git branch --track docs remotes/origin/docs || true # ignore error in case the docs branch already exists
 	git show docs:man1/git-filter-repo.1 >Documentation/man1/git-filter-repo.1
 
 Documentation/html/git-filter-repo.html:
-	mkdir -p html
+	mkdir -p Documentation/html
+	git branch --track docs remotes/origin/docs || true # ignore error in case the docs branch already exists
 	git show docs:html/git-filter-repo.html >Documentation/html/git-filter-repo.html
```

Comment:
Same comments as for above.  Also, the commit message needs some fixes:

- prefix summary with area and drop the trailing colon
- second line should be blank
- the first bullet reads a little funny.  Perhaps 'correct paths to including missing "Documentation/" prefix'?
- drop the "need to" from the second bullet (though the second bullet will change based on the other feedback)
- Include a Signed-off-by as mentioned on the other commit.